jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -142,25 +148,39 @@ public static LoggingMXBeanImpl getInstance() {
 	@Override
 	public String getLoggerLevel(String loggerName) {
```

Comment:
I haven't finished / tested this out with the java 8 code back in but does this look too confusing to read? I'm not sure if its really worth the effort to not just have the whole method for java 8 and java 9 one after another.

Reply:
This method I mean

Reply:
Works for me. This is typically how we have done it in the past to avoid duplicating code and to avoid needing multiple updates when making changes, which risks that something will be missed.

Reply:
Just pushed the rest of the changes to include the old code for Java 8